core/src/main/java/com/orange/ouds/core/component/OudsColoredBox.kt

```diff
@@ -197,8 +197,12 @@ enum class OudsColoredBoxColor {
                 SurfaceTertiary -> OudsColorKeyToken.Surface.Tertiary
             }.value
         }
+    
+    val isSupported: Boolean
```

Comment:
We can use it in the `CheckedContent` of the `OudsColoredBox` in place of `color.mode.isSupported`.